service-python/tests/assessclaim/dc7101/bp_sufficiency_test.py

```diff
@@ -814,7 +1002,7 @@
                                               'value': 200}}],
                  'twoYearsBpCount': 2}
         ),
-        (
+        (  # remove LH duplicate but keep MAS data intact
```

Comment:
test case for deduplication

Reply:
Nice to have: more test cases to validate that only "same diastolic
     value, systolic value and date" are removed as duplicate